stream_alert/rule_processor/rules_engine.py

```diff
@@ -219,6 +219,15 @@ def update(cls, results, parent_key, nested_results):
         """Update nested_results by inserting parent key to beginning of list.
         Also combine results and nested_results into one dictionary
 
+        Args:
+            results (dict): A dict of normalized_types with original key names
+            parent_key (str): Parent key of values in nested_results. The values
+                in nested_results are original keys of normalized types.
+            nested_results (dict): A dict of normalized_types from nested record
+
+        Returns:
```

Comment:
@chunyong-lin  this function doesn't actually 'return' anything. please remove this since it alters a dictionary by reference

Reply:
Good catch! Thanks!